x/gov/abci.go

```diff
@@ -5,43 +5,51 @@ import (
 	"time"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	govTypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
 
 	"github.com/shentufoundation/shentu/v2/common"
 	"github.com/shentufoundation/shentu/v2/x/gov/keeper"
-	"github.com/shentufoundation/shentu/v2/x/gov/types"
 	shieldtypes "github.com/shentufoundation/shentu/v2/x/shield/types"
 )
 
 func removeInactiveProposals(ctx sdk.Context, k keeper.Keeper) {
```

Comment:
do we need all this? I suppose we can probably wrap the existing gov endblocker

Reply:
we need these